CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -2198,6 +2198,16 @@ local function CreateChat()
 			pcall(function()
 				if sendingPlayer == PlayersService.LocalPlayer then
 					isDebugCommand = game:GetService("GuiService"):ShowStatsBasedOnInputString(chattedMessage)
+					
+					-- allows dev console to be opened on mobile
+					-- NOTE: Removed ToggleDevConsole bindable event, so engine no longer handles this
+					if string.lower(chattedMessage) == "/console" then
+						local devConsoleModule = require(RobloxGui.Modules.DeveloperConsoleModule)
```

Comment:
I will add it to the first if check then since ShowStatsBasedOnInputString() is RobloxScript only.